### PR TITLE
Fix incorrect links to source code in docs

### PR DESCRIPTION
**Pull request by GNi33**
As the ecommerce framework has been moved into a standalone bundle, some links in the documentation which are still pointing to the main pimcore repository are broken

### CODE REVIEW

File: doc/09_Working_with_Prices/09_Calculate_with_Prices.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
 As floating point numbers (`float`, `double`) are not able to represent numbers exactly (see [here](http://floating-point-gui.de/)
 if you want to know details), and exact numbers are a strict demand to e-commerce applications the E-Commerce Framrwork
-uses [`Decimal`](https://github.com/pimcore/pimcore/blob/11.x/bundles/EcommerceFrameworkBundle/Type/Decimal.php)
+uses [`Decimal`](https://github.com/pimcore/ecommerce-framework-bundle/tree/1.x/src/Type/Decimal.php)
 value objects instead of floats to represent prices. These value objects internally store the represented value as integer
 by defining a fixed amount of supported digits (so-called `scale`) after the comma and by multiplying the actual value
 with the given scale on construction. The scale is set to 4 by default, but can be changed globally in the `pimcore_ecommerce_framework.decimal_scale`
 config entry.
-
+Decimal
```
Comment (mcassier31) [inline doc/09_Working_with_Prices/09_Calculate_with_Prices.md]:
```suggestion
```

File: doc/09_Working_with_Prices/09_Calculate_with_Prices.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ use Pimcore\Bundle\EcommerceFrameworkBundle\Type\Decimal;
 var_dump(Decimal::create('123.55555', 4, PHP_ROUND_HALF_DOWN)->asString()); // 123.5555
 var_dump(Decimal::create('123.55555', 4, PHP_ROUND_HALF_UP)->asString());   // 123.5556
 ```
-
+../../
```
Comment (mcassier31) [inline doc/09_Working_with_Prices/09_Calculate_with_Prices.md]:
```suggestion
```

File: doc/10_Working_with_Availabilities.md [hunk 3]
```diff
@@ -9,10 +9,10 @@ Each product can have its own Availability System.
 
 ## Configuration of Availability Systems
 
-A availability system is a class implementing `Pimcore\Bundle\EcommerceFrameworkBundle\AvailabilitySystem\AvailabilitySystemInterface` which 
+An availability system is a class implementing `Pimcore\Bundle\EcommerceFrameworkBundle\AvailabilitySystem\AvailabilitySystemInterface` which 
 is defined as service and registered with a name in the `pimcore_ecommerce_framework.availability_systems` configuration tree. 
 
-Currently the framework ships only with a [sample implementation](https://github.com/pimcore/pimcore/blob/11.x/bundles/EcommerceFrameworkBundle/AvailabilitySystem/AvailabilitySystem.php#L20)
+Currently, the framework ships only with a [sample implementation](https://github.com/pimcore/ecommerce-framework-bundle/blob/1.x/src/AvailabilitySystem/AvailabilitySystem.php#L20)
```
Comment (mcassier31) [inline doc/10_Working_with_Availabilities.md]:
```suggestion
Currently, the framework ships only with a [sample implementation](https://github.com/pimcore/ecommerce-framework-bundle/blob/1.x/src/AvailabilitySystem/AvailabilitySystem.php#L21)
```

**Review by mcassier31 — CHANGES_REQUESTED** [review]
Hi! Thanks for your contribution!
Could you apply the following changes, please? :)

**Comment by mcassier31** [comment]
Thanks for the update! Could you also rebase to 1.0 (bugfix) please?

**Comment by GNi33** [comment]
I see, would I need to change the absolute links from `/pimcore/ecommerce-framework-bundle/blob/1.x/` to `/pimcore/ecommerce-framework-bundle/blob/1.0/` in that case?

**Comment by mcassier31** [comment]
Hi! As 1.0 will be deleted after the next minor release, keeping the links pointing to 1.x is better. In that way, we will not need to update them again for 1.1 :)

**Pull request merged**